worldedit-core/src/main/java/com/sk89q/worldedit/internal/util/ChunkDeleter.java

```diff
@@ -0,0 +1,258 @@
+/*
+ * WorldEdit, a Minecraft world manipulation toolkit
+ * Copyright (C) sk89q <http://www.sk89q.com>
+ * Copyright (C) WorldEdit team and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.sk89q.worldedit.internal.util;
+
+import com.sk89q.worldedit.math.BlockVector2;
+import com.sk89q.worldedit.world.storage.McRegionChunkStore;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.RandomAccessFile;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@SuppressWarnings({"UseOfSystemOutOrSystemErr", "CallToPrintStackTrace"})
+public class ChunkDeleter {
+    public static final Logger logger = LoggerFactory.getLogger(ChunkDeleter.class);
+
+    public static void main(String[] args) throws IOException {
+        if (args.length < 1) {
+            System.err.println("Chunk list must be specified.");
+            System.exit(1);
+            return;
+        }
+        boolean doBackup = args.length > 1 && args[1].equalsIgnoreCase("true") || promptDoBackup();
+        String chunkFile = args[0];
+
+        ChunkDeleter cd = new ChunkDeleter(doBackup);
+        if (!cd.loadChunkFile(chunkFile)) {
+            System.exit(1);
+        }
+        cd.readChunksFromFile();
+
+        System.out.print("Press [enter] to begin deletion...");
```

Comment:
Perhaps "any key" would be better? Currently that's the actual criteria.

Reply:
Need to test this probably since the last time I used Scanner (like 6 years ago?) I think I remember something about it tokenizing input, so it wouldn't work until the user types a full word. /shrug maybe tho